server/src/misc.go

```diff
@@ -423,3 +423,102 @@ func yesNoFromBoolean(option bool) string {
 	}
 	return "No"
 }
+
+func hasTableNameValidCommand(s *Session, d *CommandData, data *SpecialGameData) bool {
+	if strings.HasPrefix(d.Name, "!") {
```

Comment:
return early

Reply:
Not sure I understand your point

Reply:
the way you wrote this function is an anti-pattern.
invert the logic in the if statement and return, then outdent the rest of the function

Reply:
I didn't write any function, I just enclosed existing code within a function block, since it is used identically in two different places (table_create, table_restart). Nevertheless I'll redo

Reply:
right - it couldn't be written using return early pattern in the place it was before because there was logic following the if statement (which is no longer the case)